unlock-app/src/components/interface/ModalOverlay.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { dismissWalletCheck } from '../../actions/walletStatus'
+
+const ModalOverlay = ({ children }) => (
```

Comment:
perhaps rename to `FullScreenModal`

Reply:
Rename the file, right?

Reply:
yes. Since it contains multiple components, I might go for `FullScreenModals.js`. Up to you.